client/components/app.tsx

```diff
@@ -31,5 +35,8 @@ export const App: React.FC = () => (
       <Route path='/supplier'>
         <Supplier/>
       </Route>
+      <Route path='/dashboard'>
```

Comment:
I would nest this under `/supplier/dashboard` so that it's separated from the vending machine simulator
And you might have to move this one above `/supplier` route